hs-backend-booster/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Copyright (c) Runtime Verification, Inc. All Rights Reserved. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.runtimeverification.k</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>hs-backend-booster</artifactId>
+  <packaging>jar</packaging>
```

Comment:
This seems to cause issues with the docker build setup in CI (`dependency:go-offline` fails due to not finding a jar with these coordinates). The `haskell-backend` directory indeed had some java code that ends up in a `jar` in the artefact store but `hs-backend-booster` does not have any.
Also, the `dependencies` section below is not necessary, the `hs-backend-booster` does not have any.

https://www.baeldung.com/maven-packaging-types

Reply:
[test-docker-setup.sh.txt](https://github.com/runtimeverification/k/files/11894154/test-docker-setup.sh.txt)
FYI script that I used to test locally